javascript-loops-practice/get-student-names.js

```diff
@@ -0,0 +1,12 @@
+/* exported getStudentNames */
+
+function getStudentNames(students) {
+  var values = [];
+  var x;
+  for (var i = 0; i < students.length; i++) {
+    for (x in students[i]) {
```

Comment:
You don't need a `for in` loop here...do you, Squidward
![image](https://user-images.githubusercontent.com/32880187/96300712-deb20e00-0faa-11eb-9f30-3d13ecc7ba2e.png)